### DISCUSSION TITLE
v0.1.1

### DISCUSSION THREAD

**Discussion by rbreu** [rbreu/beeref#7] [Anncouncements]
**Changed**

* Flipping an image now happens on mouse press instead of mouse release
* About dialog points to new website beeref.org
* Menus and dialogs now have a dark style to match the optics of the canvas

**Fixed**

* Double click to zoom an item and double-clicking again should now always  correctly go back to the previous position
* The outline of the rubberband selection now stays the same size  regardless of zoom

<hr /><em>This discussion was created from the release <a href='https://github.com/rbreu/beeref/releases/tag/'>v0.1.1</a>.</em>

**Comment by LichiMan** [comment]
Hi! 
First of all, thanks a lot for this software.

I've just downloaded it and Windows has found malware on the .zip file:
![image](https://user-images.githubusercontent.com/178505/129187812-386d27d4-f81e-4a70-ad43-d998503b20d1.png)

Thanks again. Bests,
Miguel

**Comment by rbreu** [comment]
That's a false positive, which apparently can happen with programs build with PyInstaller. Not because there's anything wrong with PyInstaller or BeeRef, but probably because there are known viruses built the same way. It's best to contact the support of the virus software so that they can check and whitelist BeeRef.

**Reply by LichiMan** [reply]
Great. Thanks a lot for the clarification, Rebecca.
